internal/pkg/plugin/common/reposcaffolding/download.go

```diff
@@ -0,0 +1,22 @@
+package reposcaffolding
+
+import "github.com/devstream-io/devstream/pkg/util/github"
+
+func download(owner, repo, workpath string) error {
+	ghOption := &github.Option{
+		Owner:    owner,
```

Comment:
The owner here is `TemplateOwner=devstream-io`, it's a `github.Option.Org`, not `github.Option.Owner`